src/openrct2/CmdlineSprite.cpp

```diff
@@ -803,7 +803,7 @@ int32_t cmdline_for_sprite(const char** argv, int32_t argc)
             spriteFileEntries[spriteFileHeader.num_entries - 1].offset = spriteFileData
                 + (spriteFileHeader.total_size - bufferLength);
 
-            free(buffer);
+            delete buffer;
```

Comment:
Keep this PR just for the easy self contained changes leave this for another PR.